docs/openstack/nova_adoption.md

```diff
@@ -0,0 +1,171 @@
+          override:
+            service:
+              metadata:
+                annotations:
+                  metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
+                  metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+              spec:
+                type: LoadBalancer
+          customServiceConfig: |
+            [workarounds]
+            disable_compute_service_check_for_ffu=true
+        schedulerServiceTemplate:
+          customServiceConfig: |
+            [workarounds]
+            disable_compute_service_check_for_ffu=true
+        cellTemplates:
+          cell0:
+            conductorServiceTemplate:
+              customServiceConfig: |
+                [workarounds]
+                disable_compute_service_check_for_ffu=true
+            metadataServiceTemplate:
+              enabled: false  # Enable it to run in a cell instead of top-scope
+              override:
+                  service:
+                    metadata:
+                      annotations:
+                        metallb.universe.tf/address-pool: internalapi
+                        metallb.universe.tf/allow-shared-ip: internalapi
+                        metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+                    spec:
+                      type: LoadBalancer
+              customServiceConfig: |
+                [workarounds]
+                disable_compute_service_check_for_ffu=true
+          cell1:
+            conductorServiceTemplate:
+              customServiceConfig: |
+                [workarounds]
+                disable_compute_service_check_for_ffu=true
+  '
+  ```
+  The Metadata service is deployed top scope here. Making it running within cell(s) could be done post-adoption.
+  The local Conductor services will be started for each cell, while the superconductor runs in `cell0`.
+  Note that ``disable_compute_service_check_for_ffu`` is mandatory for all imported Nova services.
+  Bear in mind to unset ``debug.preserveJobs`` later, after all post-adoption checking is done.
+
+## Post-checks
+
+> **FIXME**: real mult-stack case: post-adoption, there should be no
+> compute hosts in the central stack for the only supported
+> multi-cell topology in NG OSP.
+> Verify the other cells databases naming (should become unchanged):
+>  ```
+>  nova_api -> nova_api
+>  nova_cell0 -> nova_cell0
+>  nova_cell1 -> nova_cell1
+>  nova -> no longer existant, or nova_cell2 (if there were instances in the central stack originally)
+>  ```
+
+* See that Nova endpoints are defined and pointing to the
+  podified FQDNs and that Nova API responds.
+
+  ```bash
+  alias openstack="oc exec -t openstackclient -- openstack"
+
+  openstack endpoint list | grep nova
+
+  # Without OpenStack client for Nova installed:
+  NOVA_PUBLIC_URL=$(openstack endpoint list -c 'Service Name' -c 'Service Type' -c URL | grep nova | grep public | awk '{ print $6; }')
+  curl "$NOVA_PUBLIC_URL"
```

Comment:
I don't think we need this option, the openstackclient pod is available in the podified deployment automatically.